lib/codegen/CreateTerm.cpp

```diff
@@ -0,0 +1,190 @@
+#include "kllvm/codegen/CreateTerm.h"
+
+#include "llvm/IRReader/IRReader.h"
+#include "llvm/IR/BasicBlock.h"
+#include "llvm/IR/Constants.h"
+#include "llvm/IR/DerivedTypes.h"
+#include "llvm/IR/Instructions.h"
+#include "llvm/Support/MemoryBuffer.h"
+#include "llvm/Support/SourceMgr.h"
+
+namespace kllvm {
+
+std::string LLVM_HEADER = R"LLVM(
```

Comment:
This might be the best way to get these declarations into the generated module, but the file also had some value as documentation of our intended memory layout. Perhaps it would be possible to `#include` the text? At least we should have a file of documentation somewhere that covers this information.